docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        "force_full": true
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_login``
```

Comment:
Along those lines, I'd suggest reconsidering the names for all of the SSH options, so that they're in line with what `ssh_config(5)` lists. For example, I'd find it straightforward if we used `ssh_identity_file` instead of `key_path` below, as this is an allusion to the `IdentityFile` option.